### PR TITLE
[RHODA][Infrastructure] - Updating Test Teardown and Test Suite Tear …

### PR DESCRIPTION
**Pull request by ikanias**
…Down

### CODE REVIEW

File: resources/keywords/suite_teardown.resource [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    Login To OpenShift CLI
+    Remove Existing DBaaSNamespaces
+    Remove Existing DBaaSProviderAccounts
+
+Tear Down The Test
+    Remove Existing DBaaSInstances
+    Logout Of OpenShift CLI
+    Close Browser
+
+
```
Comment (mrrajan) [inline resources/keywords/suite_teardown.resource:25]: Please remove the blank lines on 25, 26 and after 50

File: tests/cockroachdb_import_connect.robot [hunk 2]
```diff
@@ -4,11 +4,12 @@ Metadata            Version    0.0.1
 
 Library             SeleniumLibrary
 Resource            ../resources/keywords/deploy_application.resource
+Resource            ../resources/keywords/suite_teardown.resource
 
 Suite Setup         Set Library Search Order    SeleniumLibrary
 Suite Teardown      Tear Down The Test Suite
 Test Setup          Given The Browser Is On Openshift Home Screen
-Test Teardown       Close Browser
+Test Teardown       Tear Down The Test
```
Comment (mrrajan) [inline tests/cockroachdb_import_connect.robot:12]: IMHO we require "Test Down The Test" can be included for Admin and Developer provision test suites, where the import connect test suites might not need this step as we are not provisioning any database instances.

**Review by harshkumarRH — APPROVED** [review]
LGTM

**Comment by ikanias** [comment]
Adding an updated version to this PR.

**Pull request closed without merging**